### DISCUSSION TITLE
Is it possible to stop someone from voting?

### DISCUSSION THREAD

**Discussion by m4rocks** [snapshot-labs/snapshot#3326] [Q&A] [answered]
Please note I am new to web3, blockchain technology, etc.

Let's say someone gets a hold of some tokens and gets some voting power. The community is quite small and the bad actors' vote hurts the development of the project. Maybe the bad actor has a slightly higher voting power than the rest.

My question is: Would it be possible to prevent someone from voting?

**Comment by metrox-eth** [comment, marked as answer]
In the existing strategies there is no blacklist option.
